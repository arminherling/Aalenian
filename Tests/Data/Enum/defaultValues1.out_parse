EnumDefinitionStatement: {
    NameExpression: Values
    BlockNode(3): {
        EnumMemberDefinitionStatement: {
            NameExpression: First
        }
        EnumMemberDefinitionStatement: {
            NameExpression: Second
        }
        EnumMemberDefinitionStatement: {
            NameExpression: Third
        }
    }
}
