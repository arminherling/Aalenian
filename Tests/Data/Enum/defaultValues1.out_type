TypedEnumDefinitionStatement: {
    TypeKind: enum
    TypeName: Values
    BaseType: u8
    Fields(3): {
        TypedFieldDefinitionNode: {
            Type: u8
            Name: First
            Expression: {
                U8Literal: {
                    Type: u8
                    Value: 0
                }
            }
        }
        TypedFieldDefinitionNode: {
            Type: u8
            Name: Second
            Expression: {
                U8Literal: {
                    Type: u8
                    Value: 1
                }
            }
        }
        TypedFieldDefinitionNode: {
            Type: u8
            Name: Third
            Expression: {
                U8Literal: {
                    Type: u8
                    Value: 2
                }
            }
        }
    }
}
