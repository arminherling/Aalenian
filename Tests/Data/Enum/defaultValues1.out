EnumDefinitionStatement: {
    Name: Values
    Block(3): {
        EnumMember: {
            Name: First
        }
        EnumMember: {
            Name: Second
        }
        EnumMember: {
            Name: Third
        }
    }
}
